src/riak_cs_manifest_fsm.erl

```diff
@@ -314,8 +314,14 @@ get_and_update(RcPid, WrappedManifests, Bucket, Key) ->
         {ok, RiakObject, Manifests} ->
             case update(RcPid, Manifests, RiakObject, WrappedManifests, Bucket, Key) of
                 {ok, _, _} = Res ->
-                    maybe_backpressure_sleep(riakc_obj:value_count(RiakObject)),
-                    Res;
+                    case maybe_backpressure_sleep(riakc_obj:value_count(RiakObject)) of
+                        true ->
+                            %% Backpressure sleep is triggered. Bacause holding
+                            %% previous object can lead to more siblings, discard it.
```

Comment:
My comment recommendation is:
"Backpressure sleep has been triggered. Current manifests folded by this FSM is to be discarded to prevent unnecessary interleaving by other concurrent manifest updates while this sleep."